components/table/demo/table-test.js

```diff
@@ -153,7 +153,9 @@ class TestTable extends RtlMixin(DemoPassthroughMixin(TableWrapper, 'd2l-table-w
 				<d2l-table-col-sort-button
 					@click="${this._handleSort}"
 					?desc="${this._sortDesc}"
-					?nosort="${noSort}">${fruit}</d2l-table-col-sort-button>
+					?nosort="${noSort}">
+					${fruit}
+				</d2l-table-col-sort-button>
```

Comment:
Super nit - you can put this back to what it was before so there isn't an extra change to it in the commit history where nothing really actually changed.